exercises/two-bucket/test/test_two_bucket.c

```diff
@@ -0,0 +1,139 @@
+#include "vendor/unity.h"
+#include "../src/two_bucket.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void assert_results_match(bucket_result_t expected,
+                                 bucket_result_t actual)
+{
+   TEST_ASSERT_EQUAL(actual.possible, expected.possible);
```

Comment:
The order matters here, these tests should be `TEST_ASSERT_EQUAL(expected.whatever, actual.whatever)`

The Unity error message is "FAIL: Expected \<first argument\> Was \<second argument\>".